src/config.coffee

```diff
@@ -36,6 +36,40 @@ nconf.defaults
     params: []
   dataDir: SBS_HOME
 
+# Post-process custom files and params
+customFields = nconf.get('customFields')
+
+# Ensure array
+customFields.files = customFields.files || []
+# Always add upload_file_minidump file as first file
+customFields.files.splice(0, 0,
+  name: 'upload_file_minidump'
+  downloadAs: 'upload_file_minidump.#{id}.dmp'
```

Comment:
If we're going to template this, I think this is the wrong way to go about it. What about using handlebars templates like the views?